site/realm/data_sources/mongodb-atlas/aiidprod/submissions/rules.json

```diff
@@ -13,6 +13,17 @@
             "delete": true,
             "search": true
         },
+        {
+            "name": "incident_editor",
+            "apply_when": {
+                "%%user.custom_data.roles": "incident_editor"
+            },
+            "read": true,
+            "write": true,
+            "insert": true,
+            "delete": true,
+            "search": true
+        },
```

Comment:
Add this new rule to the wiki page https://github.com/responsible-ai-collaborative/aiid/wiki/DB-schema-rules

Reply:
Done, url whitelisted and wiki updated